docs/guides/flowing-cred.md

```diff
@@ -1,5 +1,69 @@
+
+----------
+## 💧How Cred Flows
+You can visualize Cred flowing through the graph as water flowing through an ecosystem of ponds and creeks. Cred does not accumulate in one place, but rather flows through each pond, constantly moving. It flows between contributor ponds (Cred Flowing), falls as rain (Cred minting) and evaporates from everyone's ponds over time (Cred evaporation).
+
+### 🏄🏾 Cred Flowing
+Every contributor has their own pond, which represents their Cred score. Your Cred Score is the amount of Cred that is flowing *through* your pond - the amount entering is the same amount that is simultaneously leaving, flowing to other community members. Each of your contributions is interconnected with other community contributions across different platforms. Contributions are even connected through time - today's contributions connect to contributions from yesterday, last week, etc.
+
+### ☔ Cred Minting
+If community members find one of your contributions valuable, they will **Mint**, or in other words, create, Cred, and flow it to you. This Cred  magically appears out of nowhere and flows in your direction. Cred minting is just like rain, new water falling from the sky into your pond. Keep reading to gain a deeper understanding of where Minted Cred comes from.
+
+Example:<br/>
+When someone Mints Cred by reacting to your Discord post with an emoji...
+
+- It creates a new reaction node in the graph
+- It creates an edge from the person reacting (you), to the reaction node
+- It creates an edge pointing from the reaction node to the post receiving the reaction
+  
+
+### 💡 Plug-ins
+
+The types of contributions and plugins tracked by the SourceCred algorithm are likely to increase. Currently, your contributions are tracked across the these active plugins in the following ways:
+
+
+| 💬 [Discord] | 🧵 [Discourse] | 🦠 [Github]| 🥇 [Initiatives]
+| -- | -- | -- | -- |
+|Someone gives your post any emoji 👍 💯<img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' />| Someone gives your post/reply a ❤️ | You open a pull request  | You accomplish a specific task approved by the community
+
+ Each community can also change the weights of how much Cred is Minted from each action or reaction. For example, the SourceCred community uses the SourceCred emoji <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> to denote extra value, so it Mints more Cred than other emojis. Or, if a community values GitHub pull requests but uses Discourse for down time chats, they could make it so Discourse posts/replies Mint little Cred and GitHub activity Mints more. If you'd like to learn more about the plug-ins SourceCred currently supports, check out our [plug-ins link](). ADD PLUG-INS LINK WHEN IT EXISTS
+
+
+
+### 🔥 Cred Evaporation
```

Comment:
Suggest removing Cred Evaporation section, as now covered in suggested Cred Minting section.